doc/source/cluster/kubernetes/user-guides/gcp-gke-gpu-cluster.md

```diff
@@ -57,20 +43,3 @@ gcloud container clusters get-credentials kuberay-gpu-cluster --zone us-west1-b
 ```
 
 For more details, see the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
-
-## Step 4: Install NVIDIA GPU device drivers
```

Comment:
Because our EKS doc mentions the NVIDIA driver, it would be better to mention how does GKE handles it.